src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalController.java

```diff
@@ -176,6 +176,8 @@ public ResponseEntity editPaymentAccountsByOrgId(@Valid @NotNull @RequestBody Pb
                                                      @PathVariable("orgId") @NotBlank String organisationIdentifier) {
         log.info("Received request to edit payment accounts by organisation Id...");
 
+        organisationCreationRequestValidator.validateOrganisationIdentifier(organisationIdentifier);

```

Comment:
it will be good if you put custom annotation or like @Pattern on pathvariable , req param rather than calling this method each time

Reply:
Sure, I will look into this

Reply:
I have been successful in adding @Pattern on the @PathVariable:

> @Pattern(regexp = ORGANISATION_IDENTIFIER_FORMAT_REGEX, message = ORG_ID_VALIDATION_ERROR_MESSAGE_)

Reply:
ok, check exception it throws, it should throw 400

Reply:
The tests that were written for the validator method were throwing 400. After the @Pattern change they are still throwing 400. So seems to be working fine 👍

Reply:
The changes for this request have been made in the following commit: https://github.com/hmcts/rd-professional-api/pull/444/commits/b08a8b58fd22bb7e030c661c8f0630723ca2a3f8